docs/feature_unicode.md

```diff
@@ -29,6 +29,11 @@ This is the current list of Unicode input method in QMK:
 * UC_WIN: (not recommended) Windows built-in Unicode input. To enable: create registry key under `HKEY_CURRENT_USER\Control Panel\Input Method\EnableHexNumpad` of type `REG_SZ` called `EnableHexNumpad`, set its value to 1, and reboot. This method is not recommended because of reliability and compatibility issue, use WinCompose method below instead.
 * UC_WINC: Windows Unicode input using WinCompose. Requires [WinCompose](https://github.com/samhocevar/wincompose). Works reliably under many (all?) variations of Windows.
 
+To type multiple characters for things like (ノಠ痊ಠ)ノ彡┻━┻, you can use send_unicode_hex_string() much like SEND_STRING() except you would use hex values seperated by spaces.
```

Comment:
Add a comma before `except`, and put `SEND_STRING()` and the table flip inside code quotes (``).